packages/fxa-auth-server/lib/routes/password.js

```diff
@@ -458,6 +490,101 @@ module.exports = function (
         }
       },
     },
+    {
+      // This endpoint will eventually replace '/password/forgot/send_code'
+      // below.  That is also the reason for the similarity between them.
+      method: 'POST',
+      path: '/password/forgot/send_otp',
+      options: {
+        ...PASSWORD_DOCS.PASSWORD_FORGOT_SEND_OTP_POST,
+        validate: {
+          query: isA.object({
+            service: validators.service.description(DESCRIPTION.serviceRP),
+            keys: isA.boolean().optional(),
+          }),
+          payload: isA.object({
+            email: validators
+              .email()
+              .required()
+              .description(DESCRIPTION.emailRecovery),
+            service: validators.service.description(DESCRIPTION.serviceRP),
+            metricsContext: METRICS_CONTEXT_SCHEMA,
+          }),
+        },
+      },
+      handler: async function (request) {
+        if (!config.passwordForgotOtp.enabled) {
+          throw error.featureNotEnabled();
+        }
+
+        log.begin('Password.forgotOtp', request);
+        await request.emitMetricsEvent('password.forgot.send_otp.start');
+        const email = request.payload.email;
+
+        // TODO FXA-9485
+        // await customs.check(request, email, 'passwordForgotSendOtp');
```

Comment:
If the customs check fails, do we want to emit metrics for this still?

Reply:
I want to count the limiting/blocking with statsd.  But no, logging every attempt might be too much.